AdyenNetworking/APIClient/RetryOnErrorAPIClient.swift

```diff
@@ -35,16 +38,17 @@ public final class RetryOnErrorAPIClient: APIClientProtocol {
     }
     
     /// :nodoc:
-    public func perform<R>(downloadRequest request: R, completionHandler: @escaping CompletionHandler<R.ResponseType>) where R : Request, R.ResponseType == DownloadResponse {
-        apiClient.perform(downloadRequest: request, shouldRetry: { result in
+    public func perform<R>(
```

Comment:
and this one

Reply:
This one has to stay as it implements is the base protocol with the overload.

Reply:
Ended up removing retry implementation for download API. Can be added in future when needed.